web.js

```diff
@@ -35,6 +51,10 @@ app.get('/users', function (req, res) {
 app.get('/contact', function (req, res) {
 	  renderIndex(req, res);
 	});
+app.post('/contact', function (req, res){
```

Comment:
This route should have a separate context from our web routes and it should be restful:
like '/api/contactmessages'